### ISSUE TITLE
lower case converter wrong validation

### ISSUE THREAD

**Issue by toniop99** [ramsey/conventional-commits#68]
I think this is kind a bug but I am not sure if it could be the valid implementation of a **lower case converter**

Maybe is more related to jawira/case-converter than with that package.

## Steps to reproduce
If we config the package to use ```"descriptionCase": "lower"```,  if we try to commit the following commit ```type(scope): 1.1.1 ``` to release a new version, the commit message validation will fail as it detect that the versioning is not in **lower format**

## Screenshots or output
![Image showing error](https://user-images.githubusercontent.com/35426566/198844317-720b5ceb-e0bb-40e6-a00e-4681666f8a40.png)

**Comment by toniop99** [comment]
Definitely, I think that what I am commenting does not make sense, since there would be no way to differentiate the text from other types such as the dot conversion.

Sorry for the waste of time

**Comment by ramsey** [comment]
Your comment makes sense, and this has frustrated me, too. If you can think of a good way to validate the commit message like this, let me know. I'm willing to make that change.

**Comment by toniop99** [comment]
I think we can try to validate if the description is for versioning and not apply **descriptionCaseValidator** and **descriptionEndMarkValidator** in that case. For that pupose, we can use the regex specified in the [semserver website](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) as official/recommended option.

![Description validation](https://user-images.githubusercontent.com/35426566/198874369-bf4fd642-41ce-46c6-bcf0-e5e68193d409.png)

In addition, to generalize this behavior, the following options can be added to the configuration:

- ```validateDescriptionWhenVersioningCommit ``` -> if false description will not be validated.
- ```customVersioningRegex``` -> Allow the user specify custom regex to decide if validation will be applied.


This is what comes to mind. Let me know what you think!
